doc/tor.md

```diff
@@ -211,3 +211,105 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world the Tor network is monitored or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+
+- For maximum privacy, it is preferable to disable accepting incoming connections.
```

Comment:
How would one disable incoming connections for an onion node? (`listen=0` and `listenonion=1` is not acceptable; see https://github.com/bitcoin/bitcoin/pull/20769)

Reply:
> Edit: With a proxied tor connection, that is the default. Maybe that is what is meant here?

Yes